File: Dart/gob.dart
import 'dart:io' as io;
void main() {
  print("Gob's program: y/n?");
  var x = io.stdin.readLineSync();
  if (x == "y") {
    while (true) {
      io.stdout.write("Penus ");
      io.sleep(new Duration(milliseconds:100));
    }
  }
}

File: elixir/gob.exs
defmodule Penus do
  def print() do
    IO.write "Penus "
    :timer.sleep 100
    print
  end
end

x = IO.gets "Gob's program: y/n?"
cond do
  x == "y\n" ->
    Penus.print
end
